### ISSUE TITLE
Invalid references in twisted.conch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8511]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8511|
|Type|enhancement|
|Created|2016-06-27 12:55:38Z|

Following are the invalid references in twisted.conch obtained by checking out the trunk and running

```
./bin/admin/build-apidocs . apidocs
```

```
twisted.conch.insults.text.assembleFormattedText: invalid ref to formatted
twisted.conch.insults.text.assembleFormattedText: invalid ref to twisted.conch.insults.text.attributes
twisted.conch.insults.text: invalid ref to twisted.conch.insults.text.attributes
twisted.conch.interfaces.IKnownHostEntry.matchesKey: invalid ref to twisted.conch.ssh.Key
twisted.conch.ssh.factory.SSHFactory.buildProtocol: invalid ref to twisted.conch.ssh.SSHServerTransport
twisted.conch.ssh.userauth.SSHUserAuthClient.ssh_USERAUTH_FAILURE: invalid ref to MSG_USERAUTH_FAILURE
twisted.conch.tap.makeService: invalid ref to IService
```

<details><summary>Searchable metadata</summary>

```
trac-id__8511 8511
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1467032138254568 1467032138254568
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See : https://github.com/twisted/twisted/pull/234

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 784083b06ff0eb0489058d14fac0f0edd099b7be
```
#!CommitTicketReference repository="" revision="784083b06ff0eb0489058d14fac0f0edd099b7be"
Merge 8511-acabhishek942-doc: Fix epydoc references in twisted/conch

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8511
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
